sample-apps/bundles-cart-transform/GETTING_STARTED.md

```diff
@@ -0,0 +1,260 @@
+# Shopify App Template - Ruby
```

Comment:
Are you planning to update this file in a follow up PR?

Reply:
This was the original `README` that is generated by `npm init @shopify/app@latest`. I replaced the `README` with docs specific to this app. I moved the generated `README` to `GETTING_STARTED` since it had some good general information. I can remove it, if you think it's too confusing.